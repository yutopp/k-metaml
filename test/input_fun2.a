fun f (x) = x;
fun g (x) = f x;
g