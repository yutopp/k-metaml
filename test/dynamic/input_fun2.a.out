Search results:

Solution 1:
N:Int -->
-1
V:Term -->
$fn_rec g _7 => (% $fn_rec f _3 => _3) _7
